test/error.js

```diff
@@ -1,19 +1,26 @@
 'use strict';
 
-const test = require('tap').test;
-
+const expect = require('expect');
 const gulp = require('gulp');
-const task = require('../');
+const { pipe } = require('mississippi');
 const path = require('path');
+const PluginError = require('plugin-error');
+const task = require('../');
 
 const filename = path.join(__dirname, './fixtures/pug-error.pug');
 
-test('should emit errors of pug correctly', function(t) {
-  gulp.src(filename)
-    .pipe(task()
-      .on('error', function(err) {
-        t.ok(err);
-        t.ok(err instanceof Error);
-        t.end();
-      }));
+describe('error', function() {
+  it('should emit errors of pug correctly', function(done) {
+    pipe(
+      [gulp.src(filename), task()],
+      (err) => {
+        try {
+          expect(err).toBeInstanceOf(PluginError);
+          done(null);
```

Comment:
This doesn't need an explicit `null`

Reply:
Huh, my habits to pass explicit `null` values are creeping in :smile:
I got this habit because when using callbacks I expect the first parameter to be either `instanceof Error` or `null` (but not `undefined`) but I don't always know if `undefined` is normalized to `null`.

I'll remove it.

Reply:
Fixed